packages/fx-core/src/plugins/resource/apiconnector/plugin.ts

```diff
@@ -135,11 +145,38 @@ export class ApiConnectorImpl {
     await removeFileIfExist(sampleFilePath);
   }
 
+  private getAuthConfigFromInputs(inputs: Inputs): AuthConfig {
+    let config: AuthConfig;
+    if (inputs[Constants.questionKey.apiType] === AuthType.BASIC) {
+      config = {
+        AuthType: AuthType.BASIC,
+        UserName: inputs[Constants.questionKey.apiUserName],
+        Password: inputs[Constants.questionKey.apiPassword],
```

Comment:
seems we don't get any password from user input

Reply:
It is fixed. Thanks.